src/lib.rs

```diff
@@ -145,12 +146,23 @@ impl AutoCfg {
             return Err(error::from_str("output path is not a writable directory"));
         }
 
+        // Recover the RUSTFLAGS
+        let rustflags = std::env::var("RUSTFLAGS").ok().map(|rustflags| {
+            rustflags
+                .split(' ')
+                .map(str::trim)
+                .filter(|s| !s.is_empty())
+                .map(str::to_string)
+                .collect::<Vec<String>>()
```

Comment:
Can you add a comment that this is meant to match cargo's arg-splitting?
(vs. something fancier like #11 wanted to do with `shlex`)